docs/rules/require-jsdoc.md

```diff
@@ -0,0 +1,56 @@
+# Require JSDoc comment (require-jsdoc)
+
+This rule generates warnings for nodes which do not have JSDoc comments. It considered a good practice to document the behavior of different nodes to help engineers understand the functionality of the node.
+Supported nodes:
+
+* `FunctionExpression`
```

Comment:
I ran it on my personal code, it worked fine and was helpful. If you look at the unit tests I have showcased the scenarios where it warns.